client/src/views/WorkspaceDetail.vue

```diff
@@ -85,22 +85,72 @@
+              <v-spacer />
+              <v-dialog
+                v-model="tableDialog"
+                width="700"
+              >
+                <template v-slot:activator="{ on }">
+                  <v-btn
+                    class="pl-3 pr-2"
+                    color="blue darken-2"
+                    flat
+                    v-on="on"
+                  >
+                    New Table
+                    <v-icon
+                      right
+                      size="20px"
+                    >add_circle</v-icon>
+                  </v-btn>
+                </template>
+                <v-card>
+                  <v-card>
+                    <v-card-title
+                      class="headline pb-0 pt-3"
+                      primary-title
+                    >
+                      Create Table
+                    </v-card-title>
+
+                    <v-card-text class="px-4 pt-4 pb-1">
+                      <v-layout row wrap>
+                        <v-flex>
+                          <v-text-field
+                            box
+                            v-model="newTable"
+                            label="Table name"
+                            :error-messages="tableCreationError"
+                          />
+                        </v-flex>
+                      </v-layout>
+                      <v-layout row wrap>
+                        <v-flex>
+                          <file-input @handle-file-input="handleFileInput" v-bind:types="fileTypes"/>
+                        </v-flex>
+                      </v-layout>
+                    </v-card-text>
+
+                    <v-divider></v-divider>
+
+                    <v-card-actions class="px-4 py-3">
+                      <v-spacer></v-spacer>
+                      <v-btn :disabled="tableCreateDisabled" @click="createTable">
+                        Create Table
+                      </v-btn>
+                    </v-card-actions>
+                  </v-card>
+                </v-card>
+              </v-dialog>
             </v-card-title>
 
-            <v-card-text>
-              <v-layout row wrap>
-                <v-flex>
-                  <v-text-field v-model="newTable" placeholder="name your table" solo :error-messages="tableCreationError"/>
-                </v-flex>
-              </v-layout>
-              <v-layout row wrap>
-                <v-flex>
-                  <file-input @handle-file-input="handleFileInput" v-bind:types="fileTypes"/>
-                </v-flex>
-              </v-layout>
-              <v-btn :disabled="tableCreateDisabled" @click="createTable">create table</v-btn>
+            <v-card-text class="pt-0">
+              <v-divider />
+              <div class="content pt-3">
+                <p class="grey--text text--darken-1">Tables info Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus non est et nisl egestas molestie. Morbi congue varius ipsum.</p>
```

Comment:
I need to send you copy for this.

Reply:
Based on our conversation, we're disregarding this for now.